code/modules/background/citizenship/unathi.dm

```diff
@@ -39,6 +39,7 @@
 	name = "Izweski Hegemony Consular Officer"
 
 	uniform = /obj/item/clothing/under/unathi
+	accessory = /obj/item/clothing/accessory/holster/waist
```

Comment:
All other consulars don't spawn with a holster, I see no reason to give the unathi one. Either they don't spawn with one, or all spawn with a holster. Plus, they can get it from the loadout.